deps/src/main/java/com/microsoft/azure/sdk/iot/deps/ws/impl/WebSocketImpl.java

```diff
@@ -21,7 +21,6 @@
 
 public class WebSocketImpl implements WebSocket, TransportLayer
 {
-    private final int _maxFrameSize = (4 * 1024) + (16 * WebSocketHeader.MED_HEADER_LENGTH_MASKED);
     private boolean _tail_closed = false;
```

Comment:
Just curious what is the warning if something is global like this?

Reply:
This particular one was used only in one place-  warning is "field can be local"